base/cvd/cuttlefish/host/commands/cvd/selector/creation_analyzer.cpp

```diff
@@ -85,23 +73,11 @@ static std::unordered_map<unsigned, InstanceLockFile> ConstructIdLockFileMap(
   return mapping;
 }
 
-static Result<bool> IsIdAvailable(const InstanceDatabase& instance_database,
-                                  const unsigned id) {
-  auto subset =
-      CF_EXPECT(instance_database.FindInstances(Query{kInstanceIdField, id}));
-  return subset.empty();
-}
-
 Result<std::vector<PerInstanceInfo>>
 CreationAnalyzer::AnalyzeInstanceIdsInternal(
     const std::vector<unsigned>& requested_instance_ids) {
   CF_EXPECT(!requested_instance_ids.empty(),
             "Instance IDs were specified, so should be one or more.");
-  for (const auto id : requested_instance_ids) {
-    CF_EXPECT(IsIdAvailable(instance_database_, id),
-              "instance ID #" << id << " is requested but not available.");
-  }
```

Comment:
Is this validation still happening somewhere else?

Reply:
Yes, in instace_database.cpp line 118. Creation of the new group+instances will fail because the requested instance id is in use.